pyTransition/pyTransition/transition.py

```diff
@@ -39,6 +42,46 @@ def __init__(self, url, username, password, token=None):
         else:
             self.token = self.__request_token(username, password)
 
+    @staticmethod
+    def __normalize_url(url):
+        """Normalize the URL.
+
+        Remove extra characters from the URL
+        """
+        parsed_url = urlparse(url)
+        netloc = parsed_url.netloc.lower()
+
+        if netloc.endswith(':80') and parsed_url.scheme == 'http':
+            netloc = netloc[:-3]
```

Comment:
What does this do? What's `:-3`?

Reply:
You'll have to ask chatgpt... ;)

Just kidding, it remove the extra :80 if we already specified http as protocol.